zebra-rpc/src/methods.rs

```diff
@@ -869,6 +994,22 @@ pub struct GetBlock(#[serde(with = "hex")] SerializedBlock);
 #[derive(Copy, Clone, Debug, Eq, PartialEq, serde::Deserialize, serde::Serialize)]
 pub struct GetBestBlockHash(#[serde(with = "hex")] block::Hash);
 
+/// Response to a `z_gettreestate` RPC request.
+///
+/// Contains the hex-encoded Sapling & Orchard note commitment trees, and their
+/// corresponding [`block::Hash`], [`Height`], and block time.
+#[derive(serde::Serialize)]
+pub struct GetTreestate {
+    #[serde(with = "hex")]
+    hash: block::Hash,
+    height: Height,
+    time: u32,
+    #[serde(with = "hex")]
+    sapling_tree: sapling::tree::SerializedTree,
+    #[serde(with = "hex")]
+    orchard_tree: orchard::tree::SerializedTree,
```

Comment:
Optional: can you document each field?

I have just been copying the RPC docs for each field, with extra notes about specific tricky serialization formats. (See PR #4218.)